errors/error.go

```diff
@@ -0,0 +1,10 @@
+package errors
+
+import "errors"
+
+var (
+	// ErrNoContent custom error for no record found
+	ErrNoContent = errors.New("no model exist in database")
+	// ErrConflict custom error for record already exist
+	ErrConflict = errors.New("user model is already exist in database")
```

Comment:
you have models other than user as well. this should be generic

Reply:
Done